src/classes/UTIL_OrgTelemetry_SVC.cls

```diff
@@ -194,42 +209,19 @@ public without sharing class UTIL_OrgTelemetry_SVC {
                 break;
             }
         }
-        System.FeatureManagement.setPackageBooleanValue('HasUserManagedTDTM', hasUserManagedTrigger);
+        System.FeatureManagement.setPackageBooleanValue(TelemetryParameterName.HasUserManagedTDTM.name(), hasUserManagedTrigger);
     }
 
     /**
      * @description Total number of records in the Error__c custom object
      */
     private void handleErrorLogCount() {
         try {
-            Integer count = [SELECT Count() FROM Error__c];
-            System.FeatureManagement.setPackageIntegerValue('Data_CountErrorLog', count);
-        } catch (Exception ex) {}
-    }
-
-    /**
-     * @description Total number of Opportunities associated with a Recurring Donation
-     */
-    private void handleCountOfRecurringDonationOpps() {
-        try {
-            Integer count = [SELECT Count()
-                FROM Opportunity
-                WHERE npe03__Recurring_Donation__c IN (SELECT Id FROM npe03__Recurring_Donation__c)];
-            System.FeatureManagement.setPackageIntegerValue('Data_CountRdOppsAll', count);
-        } catch (Exception ex) {}
-    }
-
-    /**
-     * @description Total number of Opportunities associated with an Open Ended Recurring Donation
-     */
-    private void handleCountOfRecurringDonationOpps_OpenEnded() {
-        String openStatusValue = System.Label.npe03.RecurringDonationOpenStatus;
-        try {
-            Integer count = [SELECT Count()
-                FROM Opportunity
-                WHERE npe03__Recurring_Donation__r.npe03__Open_Ended_Status__c = :openStatusValue];
-            System.FeatureManagement.setPackageIntegerValue('Data_CountRdOppsOpenEnded', count);
-        } catch (Exception ex) {}
+            Integer count = Database.countQuery('SELECT Count() FROM Error__c');
```

Comment:
The `Database.countQuery()` counts up to 50,000 records. If there are more, the following error is generated:
> Too many query rows: 50001 from with count()

Reply:
I just test this in the XXL org against the Opportunity object.

This query timed out: `SELECT Count() FROM Opportunity;`
While this worked fine in ExecAnon: `System.debug(Database.countQuery('SELECT Count() FROM Opportunity'));`

Reply:
I take this back, the `countQuery()` did have the limit as the SOQL query on retrieval of 50K records, however by testing total record count in a table resulted in ~133K and ~3M records, no error was raised on LDV XL org.
However, the following query executed in the ExecAnon, but in the code it is used in the batch `start()`, resulted in the > Internal Salesforce Error:
`SELECT Count() FROM Opportunity WHERE npe03__Recurring_Donation__c != null`
The previous file version had `handleCountOfRecurringDonationOpps()` that was working on the LDV orgs, should the same query be used in the batch `start()` method?

Reply:
The query handling in a queryLocator in a batch job is a bit different than the complete than the query that existed in `handleCountOfRecurringDonationOpps`. Both technically have the two minute time limit, but batch job queries are meant to be able to handle longer running queries mainly because they don't need return a full data set for the batch job to start. You can see this if you watch the ApexJobs page on a batch job with a lot of batch iterations. The number of batch iterations actually increases as the job runs because it's still compiling the data set while the job is executing. Also, I've tested the new batch job a couple of times in the XXL org and it ran perfectly both times.